modules/react/menu/lib/MenuCard.tsx

```diff
@@ -1,46 +1,47 @@
 import * as React from 'react';
 
 import {Card} from '@workday/canvas-kit-react/card';
-import {space, type} from '@workday/canvas-kit-react/tokens';
+import {type} from '@workday/canvas-kit-react/tokens';
+
 import {
-  styled,
-  TransformOrigin,
   createSubcomponent,
-  StyledType,
   ExtractProps,
   createElemPropsHook,
 } from '@workday/canvas-kit-react/common';
 import {getTransformFromPlacement} from '@workday/canvas-kit-react/popup';
+import {system} from '@workday/canvas-tokens-web';
 
 import {useMenuModel} from './useMenuModel';
+import {createStencil, calc} from '@workday/canvas-kit-styling';
+import {mergeStyles} from '../../layout';
 
 export interface MenuCardProps extends ExtractProps<typeof Card, never> {
   children?: React.ReactNode;
 }
 
-const StyledCard = styled(Card)<
-  StyledType & {width?: number | string; transformOrigin?: TransformOrigin}
->(
-  type.levels.subtext.large,
-  {
+export const menuCardStencil = createStencil({
+  vars: {
+    minWidth: '1px',
+    transformOriginVertical: 'top',
+    transformOriginHorizontal: 'left',
+  },
+  base: ({transformOriginVertical, transformOriginHorizontal, minWidth}) => ({
+    ...type.levels.subtext.large,
```

Comment:
This should probably use our `system` tokens for this. 

You'll need to add the color as well.

```suggestion
    ......system.type.subtext.large,
    color: base.blackPepper300
```